tests/Ormolu/Diff/TextSpec.hs

```diff
@@ -23,6 +23,7 @@ spec =
     stdTest "joined-hunk" "main-and-bar" "main-and-bar-v2"
     stdTest "two-hunks" "main-and-baz" "main-and-baz-v2"
     stdTest "trimming" "spaced" "spaced-v2"
+    stdTest "trailing-blank-line" "trailing-blank-line" "trailing-blank-line-v2"
```

Comment:
I suggest we rename the files to be more descriptive, so that we have:

```haskell
  stdTest "trailing-blank-line" "no-trailing-blank-line" "with-trailing-blank-line"
```

The `trailing-blank-line` output should then reflect that the trailing blank line has been added.